src/index.js

```diff
@@ -0,0 +1,203 @@
+    const up = document.getElementById('up');
+    up.addEventListener('click', increaseTemp);
+    const down = document.getElementById('down');
+    down.addEventListener('click', decreaseTemp);
+    const sky = document.getElementById('sky-select');
+    sky.addEventListener('change', changeSky);
+    const city = document.getElementById('city');
+    city.addEventListener('change', changeCity);
+    const getWeatherButton = document.getElementById('getWeatherButton');
+    getWeatherButton.addEventListener('click', getLocation);
+    const resetButton = document.getElementById('resetButton');
+    resetButton.addEventListener('click', reset);
+};
+
+document.addEventListener('DOMContentLoaded', registerEventHandlers);
+
+const convertKelvinToFahrenheit = (kelvin) => {
+    return Math.round((kelvin - 273.15) * 1.8 + 32);
+};
+
+const getLocation = () => {
+    const city = document.getElementById('cityName');
+    const cityName = city.textContent;
+    axios
+        .get('http://localhost:5000/location', {
+            params: {
+                q: cityName,
+            },
+        })
+        .then((response) => {
+            const latitude = response.data[0].lat;
+            const longitude = response.data[0].lon;
+            getWeather(latitude, longitude);
+        })
+        .catch((error) => {
+            console.log(error);
+        });
+};
+const getWeather = (latitude, longitude) => {
+    axios
+        .get('http://localhost:5000/weather', {
+            params: {
+                lat: latitude,
+                lon: longitude,
+            },
+        })
+        .then((response) => {
+            const weather = response.data;
+            state.temp = convertKelvinToFahrenheit(weather.current.temp);
+            updateTemp();
+            const skyBackground = document.getElementById('sky');
+            const skyAccessible = document.getElementById(
+                'weatherGardenAccessibleSky'
+            );
+            if (weather.current.weather[0].main === 'Clouds') {
+                state.backgroundImage = images.backgroundCloudy;
+                state.skyAccessible = accessibleText.backgroundCloudy;
+            } else if (weather.current.weather[0].main === 'Clear') {
+                state.backgroundImage = images.backgroundSunny;
+                state.backgroundImage = accessibleText.backgroundSunny;
+            } else if (weather.current.weather[0].main === 'Rain') {
+                state.backgroundImage = images.backgroundRaining;
+                state.backgroundImage = accessibleText.backgroundRaining;
+            } else if (weather.current.weather[0].main === 'Snow') {
+                state.backgroundImage = images.backgroundSnowing;
+                state.backgroundImage = accessibleText.backgroundSnowing;
+            }
+            skyBackground.style.backgroundImage = `url('${state.backgroundImage}')`;
+            skyAccessible.textContent = state.skyAccessible;
```

Comment:
This code is very close to `changeSky`. Is there a way we could rewrite `changeSky` to make it more flexible so it could be used in both circumstances?